packages/bsky/src/api/util.ts

```diff
@@ -25,3 +29,25 @@ export const clearlyBadCursor = (cursor?: string) => {
   // hallmark of v1 cursor, highly unlikely in v2 cursors based on time or rkeys
   return !!cursor?.includes('::')
 }
+
+export function isFeedPostHiddenReply({
```

Comment:
kinda wanna move this onto the `Views` class next to other checks like `viewerBlockExists` and `actorIsTakendown`

Reply:
Ah yes that's much better!